### PR TITLE
Imported examples

### PR DESCRIPTION
**Pull request by SteveWilkins1123**
﻿

Fixes #

### Changes proposed in this pull request

-
-
-

### Other links

-
-
-

### CODE REVIEW

**Review by timayabi2020 — APPROVED** [review]

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md [hunk 1]
```diff
@@ -1,18 +1,30 @@
-### Example 1: {{ Add title here }}
-```powershell
-PS C:\> {{ Add code here }}
-
-{{ Add output here }}
+###Example 1
```
Comment (peombwa) [inline src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:1]: @SteveWilkins1123, the title formatting is broken due to a missing whitespace:
```suggestion
### Example 1
```

Also, are you using a script to read the examples from the reference docs? If so, can we also include the title of the example. See example at [Get-MgDomainFederationConfiguration.md](https://github.com/msewaweru/msgraph-sdk-powershell/blob/0f1eeedd772a260ac93d75b7a98991dd30ae5d61/src/Identity.DirectoryManagement/Identity.DirectoryManagement/examples/v1.0/Get-MgDomainFederationConfiguration.md#example-1-get-the-federation-settings-for-a-federated-domain).

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
 ```
```
Comment (peombwa) [inline src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:2]: Code blocks are missing syntax highlighting for powershell.
```suggestion
``` powershell
```

**Review by peombwa — CHANGES_REQUESTED** [review]
@SteveWilkins1123, thanks for the contribution! Please see my suggestions below on the PR.

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md (hunk 1 above)
Comment (SteveWilkins1123) [reply src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:1]: That can't be done for 2 reasons. In a lot of cases the title is missing, and I am not reading them from the reference docs

File: src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md (hunk 2 above)
Comment (SteveWilkins1123) [reply src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:2]: I was asked not to include powershell

Comment (SteveWilkins1123) [reply src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md:2]: I can include it again :-)

**Comment by SteveWilkins1123** [comment]
Will resubmit new PR

**Pull request closed without merging**